lib/teleterm/api/proto/v1/service.proto

```diff
@@ -124,6 +143,69 @@ enum PasswordlessPrompt {
   PASSWORDLESS_PROMPT_CREDENTIAL = 3;
 }
 
+message GetAccessRequestRequest {
+  // cluster_uri is the cluster uri
+  string cluster_uri = 1;
+  // specifcies a specific request id
+  string id = 2;
```

Comment:
Access request RPCs are not consistent when it comes to naming the ID field in the request message. Some use `id`, some use `request_id`, others `access_request_ids`.

Since for request messages related to gateways we use `gateway_uri`, I'd recommend sticking with either `request_id` or `access_request_id`. `access_request_id` strikes me as a better choice because it's more clear.

The `id` field on the `AccessRequest` message can stay as it is IMHO as it's grouped withing a specific message.

Reply:
Thanks for keeping me accountable here. I agree with `access_request_id` 👍